### PR TITLE
LPD-35999 Added link to Cloud Console

### PR DESCRIPTION
**Pull request by welitonpe**

### CODE REVIEW

**Comment by welitonpe** [comment]
ci:test:sf

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/CustomerDashboard/pages/Apps/App/Provisioning/Types/index.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	[StepCloudInstallation.INSTALLATION]: StepsInformationProps;
+	[StepCloudInstallation.PROJECT]: StepsInformationProps;
+	[StepCloudInstallation.SUCCESS]: StepsInformationProps;
+};
+
+export type ExtendBannerProps = {
+	project: any;
+};
+
+export const LICENSE_TYPE_KEY = 'license-type';
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/CustomerDashboard/pages/Apps/App/Provisioning/Types/index.ts:44]: delete

```suggestion

```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/CustomerDashboard/pages/Apps/App/Provisioning/components/EnvironmentRadio.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import NewRadioCard from '../../../../../../../components/RadioCardList/components/RadioCard';
+
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/CustomerDashboard/pages/Apps/App/Provisioning/components/EnvironmentRadio.tsx:6]: Comes before the import

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/CustomerDashboard/pages/Apps/App/Provisioning/components/EnvironmentRadio.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+const EnvironmentRadio: React.FC<any> = ({
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/CustomerDashboard/pages/Apps/App/Provisioning/components/EnvironmentRadio.tsx:7]:
```suggestion
const EnvironmentRadio: React.FC<TYPE_ME> = ({
```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/CustomerDashboard/pages/Apps/App/Provisioning/components/EnvironmentRadio.tsx [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+	return (
+		<>
+			{selectedProject?.environments?.map(
+				(projectEnvironment: any, index: number) => {
+					const [projectName, environmentId] =
+						projectEnvironment.split('-');
+
+					return (
+						<div key={index}>
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/CustomerDashboard/pages/Apps/App/Provisioning/components/EnvironmentRadio.tsx:24]: Remove

```suggestion
```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/CustomerDashboard/pages/Apps/App/Provisioning/components/ProjectRadio.tsx [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+const ProjectRadio: React.FC<any> = ({projects, selectedProject, setValue}) => {
+	const handleSelectRadio = (selectedRadio: RadioOption<any>) => {
+		setValue('project', selectedRadio.value);
+	};
+
+	return (
+		<div>
+			{projects.map((userProject: any, index: number) => {
+				return (
+					<div key={index}>
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/CustomerDashboard/pages/Apps/App/Provisioning/components/ProjectRadio.tsx:21]: delete


```suggestion
```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/CustomerDashboard/pages/Apps/App/Provisioning/components/ProjectRadio.tsx [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+import NewRadioCard from '../../../../../../../components/RadioCardList/components/RadioCard';
+import {convertMegabyteToGigabyte} from '../../../../../../GetApp/hooks/useGetResourceInfo';
+
+const ProjectRadio: React.FC<any> = ({projects, selectedProject, setValue}) => {
+	const handleSelectRadio = (selectedRadio: RadioOption<any>) => {
+		setValue('project', selectedRadio.value);
+	};
+
+	return (
+		<div>
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/CustomerDashboard/pages/Apps/App/Provisioning/components/ProjectRadio.tsx:18]: Use fragment

```suggestion
		<>
```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/CustomerDashboard/pages/Apps/App/Provisioning/components/ProjectRadio.tsx [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayButton from '@clayui/button';
+import ClayIcon from '@clayui/icon';
+
+import NewRadioCard from '../../../../../../../components/RadioCardList/components/RadioCard';
+import {convertMegabyteToGigabyte} from '../../../../../../GetApp/hooks/useGetResourceInfo';
+
+const ProjectRadio: React.FC<any> = ({projects, selectedProject, setValue}) => {
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/CustomerDashboard/pages/Apps/App/Provisioning/components/ProjectRadio.tsx:12]:
```suggestion
const ProjectRadio: React.FC<TYPE_ME> = ({projects, selectedProject, setValue}) => {
```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/CustomerDashboard/pages/Apps/App/Provisioning/components/ProjectRadio.tsx [hunk 8]
```diff
@@ -0,0 +1,90 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayButton from '@clayui/button';
+import ClayIcon from '@clayui/icon';
+
+import NewRadioCard from '../../../../../../../components/RadioCardList/components/RadioCard';
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/CustomerDashboard/pages/Apps/App/Provisioning/components/ProjectRadio.tsx:9]:
```suggestion
import RadioCard from '../../../../../../../components/RadioCardList/components/RadioCard';
```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/CustomerDashboard/pages/Apps/App/Provisioning/components/ProvisioningTable.tsx [hunk 9]
```diff
@@ -0,0 +1,286 @@
...
+
+const ProvisioningTable: React.FC<ProvisioningTableProps> = ({
+	mutateOrder,
+	order,
+	provisioningTableData,
+	resourceRequirements,
+}) => {
+	const navigate = useNavigate();
+	const modal = useModal();
+	const unInstallModal = useModal();
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/CustomerDashboard/pages/Apps/App/Provisioning/components/ProvisioningTable.tsx:28]:
```suggestion
	const uninstallModal = useModal();
```

**Comment by kevenleone** [comment]
Just starting reviewing :)

**Pull request closed without merging**